### PR TITLE
[fuse-csi-driver] - add csi driver (fuse-csi-driver)

### PR DESCRIPTION
**Pull request by speretz-ig**
### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove irrelevant fields.]
- [ ] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### Description:
add fuse csi-driver. this driver replace the flexFuse

### CODE REVIEW

File: stable/fuse-csi-driver/templates/nodeplugin.yaml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        heritage: {{ .Release.Service }}
+    spec:
+      containers:
+        - name: nodeplugin
+          securityContext:
+            privileged: true
+            capabilities:
+              add: ["SYS_ADMIN"]
+            allowPrivilegeEscalation: true
+          image: datanode-registry.iguazio-platform.app.dev31.lab.iguazeng.com:80/iguazio/v3io-fuse:3.4.0-shayp.debug.b3558.20220306220846
```
Comment (alonmr) [inline stable/fuse-csi-driver/templates/nodeplugin.yaml]:
```suggestion
          image: {{ .Values.csi.v3ioFuse.image.repository }}:{{ .Values.csi.v3ioFuse.image.tag }}
```

File: stable/fuse-csi-driver/values.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  plugin:
+    name: fuse-csi
+    image: fuse-csi:latest
+
+    # Attempt to restore FUSE mounts. Doesn't work.
+    restoreMounts: true
+
+  registrar:
+    image: quay.io/k8scsi/csi-node-driver-registrar:v2.1.0
+
```
Comment (alonmr) [inline stable/fuse-csi-driver/values.yaml:9]:
```suggestion

    v3ioFuse:
      image:
        repository: quay.io/iguazio/v3io-fuse
        tag: latest
        pullPolicy: IfNotPresent
```

File: stable/fuse-csi-driver/templates/nodeplugin.yaml [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+          imagePullPolicy: IfNotPresent
```
Comment (alonmr) [inline stable/fuse-csi-driver/templates/nodeplugin.yaml]:
```suggestion
          imagePullPolicy: {{ .Values.csi.v3ioFuse.image.pullPolicy | quote }}
```

File: stable/fuse-csi-driver/templates/nodeplugin.yaml [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+              mountPropagation: Bidirectional
+            - name: pod-mounts
+              mountPath: /var/lib/kubelet/pods
+              mountPropagation: Bidirectional
+              volumeMounts:
+            - name: v31o-config
+              mountPath: /etc/v3io/fuse 
+              mountPropagation: Bidirectional
+            - name: v3io-log
+              mountPath: /fuse-csi-log
```
Comment (assaf758) [inline stable/fuse-csi-driver/templates/nodeplugin.yaml]: should be /fuse-log to comply with the multilog path

Comment (speretz-ig) [reply stable/fuse-csi-driver/templates/nodeplugin.yaml]: yes, i am wating to the tag from zeek to be made, and then i will push the supported commit in helm-charts

**Review by assaf758 — APPROVED** [review]

**Pull request merged**